jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarColorSchemeTests.java

```diff
@@ -44,7 +44,6 @@
     public void before() {
         shouldBeLoggedIn();
         bsPage.shouldBeOpened();
-        navbarDarkColorScheme.hover();
```

Comment:
This is the class from my previous task which I've already done. But I've just noticed that there's an odd line that does nothing in the context of that task.